README.md

```diff
@@ -129,12 +143,14 @@ network {
   mode = "bridge"
 }
 ```
-You need to install CNI plugins on nomad client nodes under `/opt/cni/bin` before you can use `bridge` networks.
+You need to install CNI plugins on Nomad client nodes under `/opt/cni/bin` before you can use `bridge` networks.
 
-  **Instructions for installing CNI plugins.**<br/>
- - $ curl -L -o cni-plugins.tgz https://github.com/containernetworking/plugins/releases/download/v0.8.1/cni-plugins-linux-amd64-v0.8.1.tgz<br/>
- - sudo mkdir -p /opt/cni/bin<br/>
- - sudo tar -C /opt/cni/bin -xzf cni-plugins.tgz
+**Instructions for installing CNI plugins.**<br/>
+```
+ $ curl -L -o cni-plugins.tgz https://github.com/containernetworking/plugins/releases/download/v0.8.1/cni-plugins-linux-amd64-v0.8.1.tgz
```

Comment:
Update that to 0.8.6 (I think that is current)

Reply:
Fixed.